app/views/items/index.html.erb

```diff
@@ -29,10 +31,15 @@
               <th><%= item.id %></th> 
               <th><a href="<%= edit_item_path item %>"><%= item.description %></a></th>   
               <% if item.current %>
```

Comment:
Im not sure about this bit when you are doing User.find you are querying the db again which is expensive. It would be much nicer if its possible to do some like item.user.name????? How else could you do this?